Source/WebKit/Platform/spi/mac/AppKitSPI.h

```diff
@@ -77,6 +77,10 @@ typedef NS_OPTIONS(NSUInteger, NSWindowShadowOptions) {
     NSWindowShadowSecondaryWindow = 0x2,
 };
 
+#if HAVE(UI_GLASS_EFFECT)
```

Comment:
This exists independently of Liquid Glass, so I don't think it needs to be guarded?